data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
+            + " In addition Bankwatch works on the People's Budget, a proposed budget for the EU that (among others) aims at ending funding for fossil fuels.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Development YES - Open Pit Mines NO",
+            link: "https://rozwojtak-odkrywkinie.pl",
+            keywords: ["coal", "grassroots", "ngo"],
+            label: "development-yes-open-pit-mines-no",
+        },
+        description: {
+            content: "This movement aims to prevent plans to build new lignite open-pit mines and transform the Polish economy to higher efficiency and renewables."
+            + " From their website its unclear if it's still active and whether the initiative has their own actions or is rather a network of other initiatives.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "REDD.plus",
+            link: "https://www.redd.plus/",
+            keywords: ["offsetting", "reforestation", "deforestation", "private-sector", "public-sector"],
+            label: "redd-dot-plus",
+        },
+        description: {
+            content: "REDD stands for Reducing emissions from deforestation and forest degradation and is part of the UN Framework Convention on Climate Change. It's a private-public initiative, seemingly setting up a registry to allow buying credits that support protection of forest areas."
+            + " The signup link doesn't work and there are [serious concerns about REDD+](https://en.wikipedia.org/wiki/Reducing_emissions_from_deforestation_and_forest_degradation#Concerns).",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Voluntary REDD+ Database",
+            link: "http://www.fao.org/forestry/vrd/data/",
+            keywords: ["offsetting", "reforestation", "deforestation", "public-sector"],
+            label: "redd-dot-plus",
+        },
+        description: {
+            content: "This registry visualizes REDD+ initiatives in members of the [Coalition of Rainforest Nations](https://www.rainforestcoalition.org)."
+            + " It shows which countries (without rainforests) are funding these initiatives. The data doesn't come with dates and seems a bit abandoned overall."
+            + " It's hosted by the FAO (Food and Agriculture Organization at the UN)."
+            + " Also there are [serious concerns about REDD+](https://en.wikipedia.org/wiki/Reducing_emissions_from_deforestation_and_forest_degradation#Concerns).",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Sandbag",
+            link: "https://sandbag.org.uk/",
+            keywords: ["legislation", "think-tank", "projection", "science", "ngo"],
+            label: "sandbag",
+        },
+        description: {
```

Comment:
same descritiption as the NGO above. Same problem with REDD

Reply:
Copy-paste error, Sandbag is unrelated to REDD. Online: `Sandbag is a think tank that lobbies governments for high-impact pro-climate legislation. Their mission is to provide targeted, evidence-based advocacy at the points of highest leverage by providing analyses and data tools They're based in London and Brussels.` (as this repo is discontinued)

Reply:
check